libmamba/src/specs/channel.cpp

```diff
@@ -45,6 +45,7 @@ namespace mamba::specs
     Channel::Channel(std::vector<CondaURL> mirror_urls, std::string display_name, platform_list platforms)
         : m_mirror_urls(std::move(mirror_urls))
         , m_display_name(std::move(display_name))
+        , m_id(m_display_name)
```

Comment:
That's still not making it a proper ID, but its something.
```suggestion
        , m_id(util::rstrip(m_display_name, '/'))
```